modules.d/99base/dracut-lib.sh

```diff
@@ -1157,8 +1157,5 @@ remove_hostonly_files() {
 # returns OK if kernel_module is loaded
 # modprobe fails if /lib/modules is not available (--no-kernel use case)
 load_fstype() {
-    if [ -z "$2" ]; then
-        set -- "$1" "$2"
```

Comment:
I think what @LaszloGombos wanted to do here was `set -- "$1" "$1"`

Reply:
Yes, I'm proposing `"${2:-"$1"}"` as more concise.